### PR TITLE
Bluetooth: controller: mark new LLCP as experimental

### PR DESCRIPTION
**Pull request by kruithofa**
Add text in Kconfig to notify users that the refactored
LLCP is experimental

Signed-off-by: Andries Kruithof <Andries.Kruithof@nordicsemi.no>

### CODE REVIEW

**Review by carlescufi — APPROVED** [review]

File: subsys/bluetooth/controller/Kconfig.ll_sw_split [hunk 1]
```diff
@@ -119,10 +119,11 @@ menu "Advanced features"
 	visible if BT_CTLR_ADVANCED_FEATURES
 
 config BT_LL_SW_SPLIT_LLCP_LEGACY
-	bool "Legacy software-based BLE Link Layer"
+	bool "Legacy software-based BLE Link Layer [EXPERIMENTAL]"
 	default y
 	help
 	  Use Zephyr legacy software BLE Link Layer ULL LLL split implementation.
+	  When not selected the experimental refactored LLCP implementation is used.
```
Comment (cvinayak) [inline subsys/bluetooth/controller/Kconfig.ll_sw_split:126]: This is LLCP related, the title and help text are incorrect.

The new refactor LLCP is to be experimental.

**Review by cvinayak — CHANGES_REQUESTED** [review]

File: subsys/bluetooth/controller/Kconfig.ll_sw_split (hunk 1 above)
Comment (thoh-ot) [reply subsys/bluetooth/controller/Kconfig.ll_sw_split:126]: This is DNM and also superseded by https://github.com/zephyrproject-rtos/zephyr/pull/39736

**Comment by kruithofa** [comment]
superseded by #39736

**Comment by kruithofa** [comment]
Superceded by #39736

**Pull request closed without merging**